examples/svm.py

```diff
@@ -110,3 +110,10 @@ def svm_from_cfg(cfg):
 inc_value = svm_from_cfg(incumbent)
 
 print("Optimized Value: %.2f" % (inc_value))
+
+
+# We can also validate our results (though this makes a lot more sense with instances)
```

Comment:
Hm, excellent question. First of all, the ML term instance (a data point) isn't the kind of instances to use in SMAC, instead one would like a cross-validation fold to be a SMAC instance. However, I'm not sure what validation on the training set should be for machine learning. Ideally, the number of CV folds is so low that all folds were evaluated during the optimization. Regarding the test set, all data points in the test set should be one SMAC instance, namely THE test instance. So, something like this:

* data points 1-70 are training data points and split into 3 SMAC instances
* data points 71-100 are one SMAC instance used for validation.